http-client/src/lib.rs

```diff
@@ -1,3 +1,12 @@
+#[cfg(not(any(feature = "tokio1", feature = "tokio02")))]
+compile_error!("Either feature \"tokio1\" or \"tokio02\" must be enabled for this crate.");
+
+#[cfg(feature = "tokio1")]
+extern crate hyper14 as hyper;
+
+#[cfg(feature = "tokio02")]
```

Comment:
```suggestion
#[cfg(all(feature = "tokio02", not(feature = "tokio1")))]
```

Typing from the top of my head so not sure if that's 100% correct, but it should avoid weird errors in case both feature flags are enabled.

Could even go as far as add another one for when both are enabled with some more human readable `compile_error!`

Reply:
I think an additional `compile_error!` is good, users will get some weird error `hyper` not found.